App/app/android/app/src/main/AndroidManifest.xml

```diff
@@ -1,5 +1,7 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.example.app">
+    <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
```

Comment:
Same as above
Please remove these changes, as it is not required to access the network state.

Reply:
do it